.github/workflows/test.yaml

```diff
@@ -36,12 +36,38 @@ jobs:
     - name: Run tests
       run: make test
     - name: Upload pytest logs on failure
-      if: ${{ failure() }} 
+      if: ${{ failure() }}
       uses: actions/upload-artifact@v2
       with:
         name: pytest-logs
         path: |
-          pytest*.log  
+          pytest*.log
+    - name: Get python version
+      run: |
+        python_version=$(python  -V | cut -d' ' -f2)
+        echo "python_version=${python_version}" >> $GITHUB_ENV
+    - name: Make version file
+      run: |
+        printf '__commit_sha__ = "09cfe03100443fb9071bba88d5c8775ff54a9ebc"\n__time__ = "2022-07-25:15:11:05"\n' > version.py
+        cp version.py "${{ github.workspace }}/app/"
+    - name: Copy site-packages in workspace
+      working-directory: /opt/hostedtoolcache/Python/${{ env.python_version }}/x64/lib/python3.9/
+      shell: bash
+      run: |
+        mkdir -p "${{ github.workspace }}/env/" && cp -fR site-packages "${{ github.workspace }}/env/"
+    - name: Install development .env file
+      working-directory: ${{ github.workspace }}
+      shell: bash
+      run: |
+        cp -f .env.example .env
+    - name: Checks for new endpoints against AWS WAF rules
```

Comment:
This seems really cool.  Think we could do a little sync on how this works?

Reply:
💯 We should talk a bit about waffles at the next dev review or whatever meeting. It's really nice.